CarbonChat-API/src/main/java/net/draycia/carbon/api/config/ChannelPings.java

```diff
@@ -31,6 +31,10 @@
     1.0F
   );
 
+  @Setting
+  @Comment("How the ping should render to the pinged player")
+  private @NonNull String display = "<bold><red><ping></red><bold>";
```

Comment:
Add a way to use <target> in the format to get just the name of the player pinged, for things like hovers and click events

Reply:
Thanks :)